### PR TITLE
chore: added mobile help.feature

### PR DESCRIPTION
**Pull request by BlueMona**
#### Relevant info and issue/PR links 
 
A Cucumber feature file for a mobile help screen. 

https://app.clubhouse.io/peerio/story/16758/mobile-simple-help-page 

#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: test/spec/settings/help.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    When  I tap "Chat"
+    Then  a chat opens with the "@support" user
+
+Scenario: I want to send logs to Peerio Support
+    When  I tap "Send" 
+    Then  an e-mail opens in the native e-mail app
+    And   all the logs are copied into the e-mail
+    And   the "to" field contains "support@peerio.com"
+    When  I tap send in the e-mail app
+    Then  I return to the "Help" screen
```
Comment (seavan) [inline test/spec/settings/help.feature]: Unfortunately, currently we cannot test native e-mail app. So we would have to stop at "an e-mail opens in the native e-mail app" confirming that external app has started.

Maybe at some point we would be able to write our own email app drop-in replacement to test it further, but we need to remove everything after "an e-mail opens in the native e-mail app" with a comment "native email check to be done"

Comment (BlueMona) [reply test/spec/settings/help.feature]: Ok, I updated this by commenting out the code you mentioned and adding the comment "native email check to be done", so we can use uncommented part for tests and still keep the commented portion as documentation.

**Comment by seavan** [comment]
@BlueMona I am going to move the cucumber scripts without backing code into test/spec-proposed because currently our CI already runs everything in test/spec. so unless it already has code and is exeucting on CI, it's gonna be in test/spec-proposed.

**Comment by BlueMona** [comment]
Ok!

**Pull request merged**